infra/image/shcontainer

```diff
@@ -32,6 +32,7 @@ container_create() {
     podman create \
            --security-opt label=disable \
            --network bridge:interface_name=eth0 \
+           --cap-add DAC_READ_SEARCH \
```

Comment:
Please have a look at line 21. This capability should be added in a different way.

Reply:
fixed.